### PR TITLE
Pet Filter Card

### PR DESCRIPTION
**Pull request by mneumark**
[Story](https://rentpath.leankit.com/card/583877373)

* created PetFilterCard
* modified how FilterCard works
  * move some of the logic down to make components that use it easier
* added support for anything to be used as a RadioGroup label

### CODE REVIEW

**Comment by michaelnease** [comment]
LGTM 👍

File: packages/react-ui-ag/src/Filters/PetFilterCard.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import themed from 'react-themed'
+import isEqual from 'lodash/isEqual'
+import { RadioGroup } from '@rentpath/react-ui-core'
+import autobind from 'autobind-decorator'
+import FilterCard from './FilterCard'
+
+@themed('*')
```
Comment (bethstevic) [inline packages/react-ui-ag/src/Filters/PetFilterCard.js]: Should this be `@themed('/^BedroomFilter/')`?

File: packages/react-ui-ag/src/Filters/PetFilterCard.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+export default class BedroomFilterCard extends Component {
+  static propTypes = {
```
Comment (bethstevic) [inline packages/react-ui-ag/src/Filters/PetFilterCard.js]: PetFilter, right?

**Comment by bethstevic** [comment]
Looks good to me 👍

**Pull request merged**